src/components/AppProvider/README.md

```diff
@@ -481,22 +481,25 @@ To provide access to your initialized Shopify App Bridge instance, we make it av
 import React from 'react';
 import {render} from 'react-dom';
 import * as PropTypes from 'prop-types';
-import {AppProvider} from '@shopify/polaris';
+import {
+  AppProvider,
+  _SECRET_INTERNAL_APP_BRIDGE_CONTEXT,
```

Comment:
You've updated the export name in the index to `AppBridgeContext` but not fixed it here.